### PR TITLE
Fetch ids in integration tests dynamically

### PR DESCRIPTION
**Pull request by loicgasser**

### CODE REVIEW

File: chsdi/tests/integration/__init__.py [hunk 1]
```diff
@@ -32,6 +35,19 @@ def tearDown(self):
...
+    def getSession(self):
+        session = scoped_session(sessionmaker())
+        yield session
+        session.close()
+
+    def getRandomFeatureId(self, bodId):
+        models = models_from_bodid(bodId)
+        with self.getSession() as session:
+            query = session.query(models[0]).limit(1)
+            reslt = query.one()
```
Comment (gjn) [inline chsdi/tests/integration/__init__.py]: Is this truly random?

Comment (loicgasser) [inline chsdi/tests/integration/__init__.py]: it's random from a user point of view, as in I don't care just give me some id. You'd need ORDER BY RANDOM() to really get a different value every time.

Comment (gjn) [inline chsdi/tests/integration/__init__.py]: Yes, I had that once. Works fine with small datasets, explodes with big datasets :=), that's why I asked.

Truly random would be better, but can live with this. Maybe there's a way to randomize the first 1000 only?

Comment (loicgasser) [inline chsdi/tests/integration/__init__.py]:
```sql
SELECT sub.id FROM (SELECT id FROM table LIMIT 1000) AS sub ORDER BY RANDOM() LIMIT 1
```

The first 1000 will always be the same, and we choose randomly one out of a 1000

Comment (gjn) [inline chsdi/tests/integration/__init__.py]: Could be worth it. But as said, not major IMO.

Comment (loicgasser) [inline chsdi/tests/integration/__init__.py]: Sure I can do it, we need to change all our tests where we have hardcoded ids in the integration tests in the future as well. Let's make it 500, build is long enough already.

**Comment by loicgasser** [comment]
I'll megre for Jenkins tonight and change that tomorrow.

File: chsdi/tests/integration/__init__.py (hunk 1 above)
Comment (gjn) [inline chsdi/tests/integration/__init__.py]: +1

**Pull request merged**